src/components/InlineCheckbox/InlineCheckbox.js

```diff
@@ -87,6 +92,7 @@ export default class InlineCheckbox extends React.Component {
       ref: this.handleRef,
       checked: false,
       disabled,
+      onChange: onChange
```

Comment:
small nitpick: we can probably use the object property value shorthand to remain consistent with the rest of the codebase